WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ValidationErrorRow.swift

```diff
@@ -3,7 +3,17 @@ import SwiftUI
 /// Renders a row for validation error with error message in red.
 ///
 struct ValidationErrorRow: View {
-    let errorMessage: String
+    private let errorMessage: String
+    private let minHeight: CGFloat
+    private let horizontalPadding: CGFloat
```

Comment:
Looks like this padding is not used in the view, which is why the error text has the extra padding as I reported yesterday.

Reply:
Fixed this in https://github.com/woocommerce/woocommerce-ios/pull/11310